app/assets/stylesheets/styles/components/guides/components/_small_screen_show_top.css.scss

```diff
@@ -0,0 +1,17 @@
+.small-guide {
+  .canopy {
+    height: 400px;
+  }
+
+  .guide-name {
+    @include background-gradient(rgba(0,0,0,0.85), rgba(0,0,0,0));
+    bottom: -408px;
+    color: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.